cms/djangoapps/contentstore/views/assets.py

```diff
@@ -30,6 +30,13 @@
 
 # pylint: disable=unused-argument
 
+request_defaults = {
```

Comment:
Nit: could you ALL_CAPS this variable, to indicate its constant-within-the-file status?